frontend/src/metabase/common/hooks/use-has-models.tsx

```diff
@@ -0,0 +1,16 @@
+import type { SearchRequest } from "metabase-types/api";
+
+import { useFetchModels } from "./use-fetch-models";
+
+export const useHasModels = (req: Partial<SearchRequest> = {}) => {
+  const modelsResult = useFetchModels({
+    limit: 1,
+    model_ancestors: false,
+    ...req,
+  });
+  const modelsLength = modelsResult.data?.data.length;
+  if (modelsLength === undefined) {
+    return undefined;
+  }
```

Comment:
will this be undefined? won't it be null?

Reply:
My tsserver is showing me:
<img width="475" alt="image" src="https://github.com/metabase/metabase/assets/130925/c4890afa-6c47-40f0-8d68-2f41be292fe6">